22_pipeslab/pipeslab.txt

```diff
@@ -3,19 +3,42 @@ For each of the following, type a single command line that performs:
   1. Print the number of lines from "ls -l" which represent a path that has execute permission to their owner.
   Answer: 
 
+  tocode@tocode-VirtualBox:~$ ls -alF | grep tocode | cut -c4 | grep x | wc | cut -d' ' -f6
+  20
+  grep x is not must
+  
   2. Print the number of lines from "ls -l" which represent a path that has execute permission to their owner, group and others.
   Answer: 
-
+ 
+  ls -alF | grep tocode | grep x..x..x | wc | cut -d' ' -f6
```

Comment:
Same issue with cut and wc.. the grep thing is actually very clever! 

Just a tiny note about ```grep```:
I recommend always using quotes with grep patterns, here it doesn't matter, but if you if pass in something that looks like a shell glob it will be expended before the command. And that's not something that you usually want.